src/formatDate.js

```diff
@@ -50,7 +50,52 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newDate = new Array(toFormat.length - 1);
+  let divider = '';
```

Comment:
```suggestion
  let divider = fromFormat[fromFormat.length - 1];
```
or something like that instead of creating cycle